topics/ecology/tutorials/openrefine_gbif/tutorial.md

```diff
@@ -251,7 +252,7 @@ kind of text, numbers and dates.
 
 > <hands-on-title> {% icon hands_on %} Basic clustering </hands-on-title>
```

Comment:
This will fix the issue with the box headers that you mentioned, somebody tried to add an icon there but that does not work. So remove this everywhere you see it to fix that. Thanks for noticing!

```suggestion
> <hands-on-title> Basic clustering </hands-on-title>
```